libs/datamodel/connectors/dml/src/field.rs

```diff
@@ -27,41 +27,37 @@ impl FieldArity {
     }
 }
 
-// TODO: when progressing with the native types implementation we should consider merging the variants `NativeType` and `Base`
-//Agreed
 /// Datamodel field type.
 #[derive(Debug, PartialEq, Clone)]
 pub enum FieldType {
     /// This is an enum field, with an enum of the given name.
     Enum(String),
     /// This is a relation field.
     Relation(RelationInfo),
-    /// native field type.
-    NativeType(ScalarType, NativeTypeInstance),
     /// This is a field with an unsupported datatype. The content is the db's description of the type, it should enable migrate to create the type.
     Unsupported(String),
     /// The option is Some(x) if the scalar type is based upon a type alias.
```

Comment:
Maybe adjust the comment since there are now two options.

Reply:
Good point — I just pushed a new doc comment.